api/envoy/config/core/v3/config_source.proto

```diff
@@ -152,7 +152,8 @@ message RateLimitSettings {
   google.protobuf.UInt32Value max_tokens = 1;
 
   // Rate at which tokens will be filled per second. If not set, a default fill rate of 10 tokens
-  // per second will be used.
+  // per second will be used. The minimal fill rate is once per year. Lower
+  // fill rates will be set to once per year.
```

Comment:
Is there a situation we want to use a zero value to just disable fill altogether? I.e. as a block result?

Reply:
Hmm.. that's an interesting take, but somewhat out of scope of this PR.
Looking at #4718 (for which this was introduced), it seems this was added to "protect" Envoy against an xDS server that may send frequent updates.
As mentioned in that thread, there are probably better ways to do so, but I don't think that setting 0 as a special value helps in achieving that.
LMK if you think otherwise.

Reply:
I see, this is xDS only, I think I was confused by the message reference that seemed more generic, `envoy_v3_api_field_config.core.v3.RateLimitSettings.fill_rate`. LGTM.